docker/android-sdk/tools/watch

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+inotifywait -mr $HOME/.android -e create -e moved_to -e modify -e attrib |
```

Comment:
@odra Could we force the builds to write these files to their own file system as opposed to the volume which they mount?